src/utils/analytics.js

```diff
@@ -38,7 +39,9 @@ function mapAuth0UserToSegmentUser(auth0user) {
       avatar: auth0user.picture,
       created_at: auth0user.created_at,
       timezone: 'Asia/Jerusalem',
-      environment: process.env.NODE_ENV
+      environment: process.env.NODE_ENV,
+      listing_id: user_metadata.listing_id,
```

Comment:
I'm not sure I completely understand, but users can have multiple listings.

Also, WDYT about having a separate method for reporting user data? listing_id should be private IMO.
It will also help us separate internal dorbel data and public profiles.